### PR TITLE
Updates in the behaviour of accessing already curated recipes on the …

### PR DESCRIPTION
**Pull request by shtukas**
…curation URL.

This change:

1. Makes the header of the read only page better (this page was not accessible to users until now)
2. Add a user event on the access of the read only page
3. Implements the logic for preventing the curation page to be re-accessed outside Ready or Pending status.

### CODE REVIEW

**Comment by shtukas** [comment]
New top page of read only page:

read-only

File: ui/app/com/gu/recipeasy/views/recipe.scala.html [hunk 1]
```diff
@@ -3,28 +3,38 @@
...
 @link = @{ "https://www.theguardian.com/" + articleId }
 @pageTopMessage(status: RecipeStatus) = @{
-    status match {
-        case New        => "Creation, pass 1 of 3."
-        case Pending    => "Creation, pass 1 of 3."
-        case Curated    => "Verification, pass 2 of 3."
-        case Verified   => "Final Check, pass 3 of 3."
-        case Finalised  => "Final Check, pass 3 of 3."
-        case _ => ""
+    if(isEditable){
```
Comment (LATaylor-guardian) [inline ui/app/com/gu/recipeasy/views/recipe.scala.html]: FYI it's encouraged to have a space between `if` and the first bracket and a space between the last bracket and the brace. The same applies with else below. But it's a small thing, don't feel the need to change it in this PR.

**Comment by LATaylor-guardian** [comment]
Looks fine to me 👍

**Comment by tomrf1** [comment]
👍

File: ui/app/com/gu/recipeasy/views/recipe.scala.html (hunk 1 above)
Comment (shtukas) [inline ui/app/com/gu/recipeasy/views/recipe.scala.html]: Ok. Cool. I will remember that :)

**Pull request merged**